### ISSUE TITLE
Add more dataset to a custom object detector weight file

### ISSUE THREAD

**Issue by josebenitezg** [AlexeyAB/darknet#4756]
Hi,

I create my own model with your repo and and I have good results with camara A, next I try the same with camera B and the results looks quite diferent. 
So I took more image from camara B and I want to add those image as dataset in my model.

How can I do that? I just can use my model as a pretrained model?

**Comment by sambo55** [comment]
Yes if you just add the data to your data folder and update your train.txt and test.txt you can continue training using your saved weights.

You might have to use -clear 1 in your training command if you've trained the weights to max_steps before.
